.circleci/config.yml

```diff
@@ -171,128 +175,97 @@ workflows:
-          cxx: clang++-4.0
-          python-version: "3.6"
+          name: clang8-py38
+          image: clang8_py38
+          cc: clang-8
+          cxx: clang++-8
+          python-version: "3.8"
 
       - build_and_test:
-          name: gcc5-py35
-          image: ubuntu16.04
-          cc: gcc-5
-          cxx: g++-5
-          python-version: "3.5"
+          name: clang7-py38
+          image: clang7_py38
+          cc: clang-7
+          cxx: clang++-7
+          python-version: "3.8"
 
       - build_and_test:
-          name: clang38-py35
-          image: ubuntu16.04
-          cc: clang-3.8
-          cxx: clang++-3.8
-          python-version: "3.5"
-
-      - pypi-linux-wheels:
-          name: wheel-build-cp27m
-          python: cp27-cp27m
-          run-tests: false
-
-      - pypi-linux-wheels:
-          name: wheel-build-cp27mu
-          python: cp27-cp27mu
-          run-tests: false
-
-      - pypi-linux-wheels:
-          name: wheel-build-cp35m
-          python: cp35-cp35m
+          name: clang6-py37
+          image: clang6_py37
+          cc: clang-6.0
+          cxx: clang++-6.0
+          python-version: "3.7"
 
       - pypi-linux-wheels:
           name: wheel-build-cp36m
           python: cp36-cp36m
+          image: manylinux1_x86_64
 
       - pypi-linux-wheels:
           name: wheel-build-cp37m
           python: cp37-cp37m
           build-sdist: true
+          image: manylinux2010_x86_64
 
-  deploy:
-    jobs:
       - pypi-linux-wheels:
-          name: wheel-deploy-cp27m
-          python: cp27-cp27m
-          run-tests: false
-          filters:
-            tags:
-              only: /^v.*/
-            branches:
-              ignore: /.*/
-
-      - pypi-linux-wheels:
-          name: wheel-deploy-cp27mu
-          python: cp27-cp27mu
-          run-tests: false
-          filters:
-            tags:
-              only: /^v.*/
-            branches:
-              ignore: /.*/
+          name: wheel-build-cp38
+          python: cp38-cp38
+          image: manylinux2010_x86_64
 
+  deploy:
+    jobs:
       - pypi-linux-wheels:
-          name: wheel-deploy-cp35m
-          python: cp35-cp35m
+          name: wheel-deploy-cp36m
+          python: cp36-cp36m
+          image: manylinux1_x86_64_x86_64
           filters:
             tags:
               only: /^v.*/
             branches:
               ignore: /.*/
 
       - pypi-linux-wheels:
-          name: wheel-deploy-cp36m
-          python: cp36-cp36m
+          name: wheel-deploy-cp37m
+          python: cp37-cp37m
+          image: manylinux2010_x86_64
+          build-sdist: true
           filters:
             tags:
               only: /^v.*/
             branches:
               ignore: /.*/
 
       - pypi-linux-wheels:
-          name: wheel-deploy-cp37m
-          python: cp37-cp37m
-          build-sdist: true
+          name: wheel-deploy-cp38
+          python: cp38-cp38
+          image: manylinux2010_x86_64
```

Comment:
Missing cp39? 👀

Reply:
Building the particular numpy version we're using from source was failing there so I decided to deal with that later, instead of finding the lowest numpy that supports 3.9 and messing with the config more than I already was. I imagine freud has to deal with this issue; I'll probably get around to it next week.

Reply:
Ah, okay. You can try https://pypi.org/project/oldest-supported-numpy/ , we use 1.14.6 for freud. https://github.com/glotzerlab/freud/blob/b890f5a3db84a13d40486736387cb527cd559508/.circleci/deploy-linux.sh#L49